utils/rpms/daos.spec

```diff
@@ -168,18 +170,23 @@ Requires: mercury >= %{mercury_version}
 Requires: libfabric >= %{libfabric_version}, libfabric < %{libfabric_max_version}
 %if (0%{?rhel} >= 8)
 Requires: fuse3 >= 3
+Requires: capstone >= 4.0.1
 %else
 Requires: fuse3 >= 3.4.2
+Requires: capstone >= 4.0.1
 %endif
 %if (0%{?suse_version} >= 1500)
 Requires: libfuse3-3 >= 3.4.2
+Requires: libcapstone-devel >= 4.0.1
```

Comment:
Does this need the devel version?

Reply:
Per the comment in the rpmlintrc file, unversioned libraries should only be used for building software.  Run-time use should use versioned shared libraries.  The use of `libcapstone-devel` here is actually resulting in it's dependency `libcapstone4` being installed, which is the only reason this appears to work.

But in actuallity, shared library RPM dependencies are auto-determined, so you don't need any `Requires: libcapstone*` at all.  RPM will include that for you implicitly.

```suggestion
```